src/index.html

```diff
@@ -8,6 +8,33 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+      <div class="card__image-container">
+        <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+        >
```

Comment:
apply code styling recommendations
```suggestion
        <img
          class="card__image"
          src="images/imac.jpeg"
          alt="imac"
        >
```